config/webpack/dev.js

```diff
@@ -1,20 +1,22 @@
+var fs = require('fs');
 var path = require('path');
 var webpack = require('webpack');
 var postcssAssets = require('postcss-assets');
 var postcssNext = require('postcss-cssnext');
 var stylelint = require('stylelint');
 var ManifestPlugin = require('webpack-manifest-plugin');
 
+const {
```

Comment:
I cherry picked it and merged it now on develop https://github.com/barbar/vortigern/commit/7961565a93571aea94517d9804e49a489d0a4a8b